editor/editor_run.cpp

```diff
@@ -258,6 +258,11 @@ Error EditorRun::run(const String &p_scene, const String &p_write_movie) {
 		}
 	}
 
+	// Pass the debugger stop shortcut to the running instance(s).
+	String shortcut;
+	VariantWriter::write_to_string(ED_GET_SHORTCUT("editor/stop"), shortcut);
+	OS::get_singleton()->set_environment("__EDITOR_STOP_SHORTCUT__", shortcut);
```

Comment:
I'd name it `GODOT_EDITOR_STOP_SHORTCUT` so it's clear it's about Godot (it's *mostly* transparent to users but if they e.g. print the `env` from their running instance, it's better if it's explicit IMO).